### PR TITLE
Tightened up the Mission section in About.

### PR DESCRIPTION
**Pull request by craigfisk**

### CODE REVIEW

**Comment by jimtyhurst** [comment]
Overlaps with #148. We need to reconcile these two pull requests.

File: src/pages/about.js [hunk 1]
```diff
@@ -15,56 +15,40 @@ const About = () => (
...
-              <li>
-                Make it easier to build stronger relationships, rather than
-                accumulating a list of contacts.
-              </li>
-              <li>
-                Provide resources for freelancers to improve their networking
-                skills.
+                We want to make it easier to stay up-to-date with the latest
+                technologies and techniques in fields closely related to your
+                specializations by connecting with other freelancers.
```
Comment (jimtyhurst) [inline src/pages/about.js:28]: This "stay up-to-date" paragraph is _not_ included in PR #148. The other text in this "Mission" section is covered in PR #148 "What's in it for you?" section.

**Review by jimtyhurst — COMMENTED** [review]
This PR is very similar to PR #148. One difference is that #148 uses a flat set of bullet points, whereas this PR preserves the hierarchical structure of the original code.

**Comment by craigfisk** [comment]
Closing, as per email.

**Pull request closed without merging**